src/infrastructure/persistence/inmemory/InMemoryResponseRepositoryAdapter.js

```diff
@@ -6,7 +6,7 @@ class InMemoryResponseRepositoryAdapter extends ResponseRepositoryPort {
     this.responses = new Map();
   }
 
-  save(response) {
+  async save(response) {
```

Comment:
i think async is not necessary here

Reply:
@przemyslawjanpietrzak  it's because we want to have consistent interface between every RepositoryAdapters, from which some needs to have asynchronous requests. If you know better solution I would be happy to hear :)